helm/reana/templates/reana-server.yaml

```diff
@@ -54,9 +54,10 @@ spec:
           - mountPath: {{ .Values.shared_storage.shared_volume_mount_path }}
             name: reana-shared-volume
           {{- range $workspace_paths := .Values.workspaces.paths }}
-          {{- if not (eq $workspace_paths "/var/reana")}}
-          - name: {{ $workspace_paths | quote |replace "/" "" }}
-            mountPath: {{ $workspace_paths }}
+          {{- $workspace_path := split ":" $workspace_paths }}
+          {{- if and (ne $workspace_path._0 $.Values.shared_storage.shared_volume_mount_path) (ne $workspace_path._1 $.Values.shared_storage.shared_volume_mount_path)}}
```

Comment:
Minor: I guess it should be:
```{{- if and (ne $workspace_path._0 $.Values.shared_storage.hostpath.root_path) (ne $workspace_path._1 $.Values.shared_storage.shared_volume_mount_path)}}```

We should compare first value with `.Values.shared_storage.hostpath.root_path` in case it will be changed

Reply:
I agree with the `$.Values.shared_storage.hostpath.root_path` comparision, but only comparing one them, might lead to try to mount a host path without its volume mount or viceversa